deployments/karnot/deployment-config.yaml

```diff
@@ -1,54 +1,70 @@
 environments:
-  - project: sepolia
-    id: aeae6c43-dc4c-42d5-951b-489c49b48502
+  - project: "Kakarot-Sepolia Starknet Staging"
```

Comment:
does it need to be the exact name?

Reply:
No, it's for your reference you can just keep it staging